spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /users/732/posts' do
```

Comment:
- [ ] **[OPTIONAL]** I would advise you to use a more descriptive text here to describe what you are testing.

  ```
  describe GET #index
  ```